tot/experiment.py

```diff
@@ -53,16 +53,52 @@ def __post_init__(self):
             data_params["freq"] = self.data.freq
         self.params.update({"_data_params": data_params})
         if not hasattr(self, "experiment_name") or self.experiment_name is None:
-            self.experiment_name = "{}_{}{}".format(
-                self.data.name,
-                self.model_class.model_name,
-                r"".join([r"_{0}_{1}".format(k, v) for k, v in self.params.items()])
-                .replace("'", "")
-                .replace(":", "_")
-                .replace("{", "_")
-                .replace("}", "_")
-                .replace("[", "_")
-                .replace("]", "_"),
+            # 1. Reduce and abbreviate df name (3 chars max)
```

Comment:
If would say we can leave out the comment here, since your proposed line of code is clear to read